src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -33,10 +34,31 @@
             :dismiss-self #(swap! state dissoc :show-perms-overlay?)}])
         [comps/SidebarButton {:style :light :margin :top :color :button-blue
                               :text "Permissions..." :icon :gear
-                              :onClick #(swap! state assoc :show-perms-overlay? true)}]])
-
+                              :onClick #(swap! state assoc :show-perms-overlay? true)}]
+        [comps/SidebarButton
+         {:style :light :margin :top :color :exception-red
+          :text "Redact" :icon :trash-can
+          :onClick
+          (fn []
+            (let [conf (js/confirm "Are you sure ?")]
+              (when conf
+                (let [type (if (contains? entity "method")
+                             "configuration" "method")
```

Comment:
First of all, there's already a `(let [config? (contains? entity "method")]` above.  Second, `create-import-form` is being called from two places that know if the entity is a method or config, so we should just pass that info to the function.